README.md

```diff
@@ -1 +1,215 @@
-# Metricswebapp
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+ 
+
+# CRYPTOIST
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [CRYPTOIST] <a name="about-project"></a>
+
+
+
+
+**[CRYPTOIST]** This project is a SPA built around coingeco's Api; it fetches the crypto market's data and displays it to the user for informative purposes in well-structured and interactive UI with help of Redux toolkit  builtin  middle ware Asyncthunk .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML5
+- CSS3
+- JS ES6
+- React
+- Redux Toolkit 
+- Redux toolkit Middleware Asyncthunk
+- Jest 
+- CoinGeco API.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[SPA]**
+- **[Redux Toolkit]**
+- **[Redux MiddleWare Asyncthunk]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] Kindly consider making a video presenting the features of the project and ensure to highlight some interesting piece of code or something you built that you are very proud of.